python/cuml/CMakeLists.txt

```diff
@@ -98,6 +105,17 @@ if(NOT ${CUML_CPU})
 
     add_subdirectory(${CUML_CPP_SRC} cuml-cpp EXCLUDE_FROM_ALL)
 
+    if(NOT CUDA_STATIC_MATH_LIBRARIES)
```

Comment:
I would have guessed that DLFW builds do not use static math libraries, but also should not have these rpath changes since it does not use CUDA wheels. Is that correct? Does this mean that dynamically linked CUDA libs will always prefer CUDA wheels over system installations, if wheels are present?

Reply:
Yes, it does. We can add an option to change it though.

Reply:
I don't think it's a big deal, unless these RPATH changes preclude the system installations from being found. DLFW will always use the system install, but the RPATH is harmless unless there's a mix of the system install and some number of library wheels. DLFW would probably need to provide stub packages for the library wheels, or otherwise patch out the library wheel dependencies, so that they wouldn't be accidentally installed.

Reply:
> unless these RPATH changes preclude the system installations from being found.

They do not.

> DLFW would probably need to provide stub packages for the library wheels, or otherwise patch out the library wheel dependencies, so that they wouldn't be accidentally installed.

It might be easier and more correct to just add a CMake option to disable the RPATHs.

Reply:
Please see my latest change. I've added a `USE_CUDA_MATH_WHEELS` option, enabled by default, but can be disabled in DLFW.